{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/2846", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/2846/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/2846/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/2846/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/2846", "id": 1679047687, "node_id": "I_kwDOJKSTjM5kFDwH", "number": 2846, "title": "Error: cannot identify image file <_io.BytesIO object at 0x7fc663ff9710>", "user": {"login": "doctorerre", "id": 79534115, "node_id": "MDQ6VXNlcjc5NTM0MTE1", "avatar_url": "https://avatars.githubusercontent.com/u/79534115?v=4", "gravatar_id": "", "url": "https://api.github.com/users/doctorerre", "html_url": "https://github.com/doctorerre", "followers_url": "https://api.github.com/users/doctorerre/followers", "following_url": "https://api.github.com/users/doctorerre/following{/other_user}", "gists_url": "https://api.github.com/users/doctorerre/gists{/gist_id}", "starred_url": "https://api.github.com/users/doctorerre/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/doctorerre/subscriptions", "organizations_url": "https://api.github.com/users/doctorerre/orgs", "repos_url": "https://api.github.com/users/doctorerre/repos", "events_url": "https://api.github.com/users/doctorerre/events{/privacy}", "received_events_url": "https://api.github.com/users/doctorerre/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 5272676193, "node_id": "LA_kwDOJKSTjM8AAAABOkanYQ", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/bug", "name": "bug", "color": "d73a4a", "default": true, "description": "Something isn't working"}], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 2, "created_at": "2023-04-21T20:29:45Z", "updated_at": "2023-04-23T15:25:53Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "Has anyone been through this?\r\n\r\nUsing Kali Linux 2023.1 and stable version\r\n\r\nenv conf:\r\n\r\n### OPEN AI\r\n## IMAGE_PROVIDER - Image provider (Example: dalle)\r\n## IMAGE_SIZE - Image size (Example: 256)\r\n##   DALLE: 256, 512, 1024\r\nIMAGE_PROVIDER=huggingface\r\nIMAGE_SIZE=256\r\n\r\n### HUGGINGFACE\r\n## HUGGINGFACE_IMAGE_MODEL - Text-to-image model from Huggingface (Default: CompVis/stable-diffusion-v1-4)\r\n## HUGGINGFACE_API_TOKEN - HuggingFace API token (Example: my-huggingface-api-token)\r\nHUGGINGFACE_IMAGE_MODEL=CompVis/stable-diffusion-v1-4\r\nHUGGINGFACE_API_TOKEN=\r\n\r\nfile attached\r\n![ASDFFFFFFFF](https://user-images.githubusercontent.com/79534115/233728861-61b43741-d67e-42f9-9bfa-a80515006e26.jpg)\r\n\r\n\r\n\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/2846/reactions", "total_count": 1, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 1}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/2846/timeline", "performed_via_github_app": null, "state_reason": null}